src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -19,11 +19,11 @@ private val LOG = logger<AmplitudeExperimentService>()
 class AmplitudeExperimentService : Disposable {
     companion object {
         const val CHANGE_AUTHENTICATE_BUTTON = "ide-jetbrains-change-authenticate-button-and-first-time-workflow"
-        const val TEST_GROUP = "TEST"
+        const val TEST_GROUP = "treatment"
+        private val storage: ConcurrentHashMap<String, Variant> = ConcurrentHashMap()
```

Comment:
Out of curiosity - what's the semantic difference between having storage outside and inside companion object? 👀

Reply:
Static storage vs local storage

Reply:
thank you!